.gitignore

```diff
@@ -1,5 +1,6 @@
 # PROJECT SPECIFIC
 postgres-data/
```

Comment:
A suggestion is to use docker volumes instead of mount volumes if possible.
Advantages:
- Better performance
- No driver required (Can have issues on Mac and Windows)
- Easy to mount to other containers (easy to backup)

Reply:
Thanks. Done.